test/filter_plugin.c

```diff
@@ -973,7 +973,11 @@ test_path_api_calls(void)
 
     if (H5PLsize(&n_paths) < 0)
         TEST_ERROR;
-    if (n_paths != 2)
```

Comment:
This is brittle and the Autotools tests now set three paths (CMake still sets two), so now I just check that at least one plugin path exists.

Reply:
Since the next check is to remove all paths, I think this 'if' should be (n_paths > 1)

Reply:
One path is fine